week-2/src/main/java/com/aston/trainee/mapper/AuthorMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.aston.trainee.mapper;
+
+import com.aston.trainee.dto.AuthorDto;
+import com.aston.trainee.entity.Author;
+import lombok.experimental.UtilityClass;
+
+import java.util.List;
+
+@UtilityClass
+public class AuthorMapper {
+    public Author fromAuthorDto(Long id, AuthorDto authorDto) {
```

Comment:
```suggestion
    public Author fromAuthorDto(AuthorDto authorDto) {
```
лучше Long id сразу засетить в AuthorDto